setup.py

```diff
@@ -1,7 +1,6 @@
 from setuptools import setup, find_packages
 
 dependencies = [
-    'swsssdk>=1.3.0',
```

Comment:
Remove dependency, keep this line will cause following error in build-image:
2022-07-18T07:28:33.7302099Z Processing ./sonic_d-2.0.0-py3-none-any.whl
2022-07-18T07:28:33.7302851Z [91mERROR: Could not find a version that satisfies the requirement swsssdk>=1.3.0 (from sonic-d) (from versions: none)